AppKit/CPSearchField.j

```diff
@@ -73,7 +73,8 @@ var CPAutosavedRecentsChangedNotification = @"CPAutosavedRecentsChangedNotificat
             @"image-cancel": [CPNull null],
             @"image-cancel-pressed": [CPNull null],
             @"image-search-inset" : CGInsetMake(0, 0, 0, 5),
-            @"image-cancel-inset" : CGInsetMake(0, 5, 0, 0)
+            @"image-cancel-inset" : CGInsetMake(0, 5, 0, 0),
+            @"search-menu-offset": CGSizeMake(10, -4)
```

Comment:
Can you check if the concept of "size offset" already exist in cappuccino or cocoa. If not, we should get the same result with an inset.

Reply:
Well, this may be my fault. I think this could be CGPointMake instead of CGSizeMake. I used CGSizeMake as I think of offset width (for horizontal offset) and offset height (for vertical offset) but this may be only in my head... What do you think ?

Reply:
if it represents a distance between a location and some origin, a CGPoint seems more appropriate.